data/Localization/german_(germany)/global.ini

```diff
@@ -42983,23 +42983,23 @@ contract_timed=~mission(Contractor|Timed)
 contract_title=~mission(Title)
 covalex_UGF_collect_desc_001=Hallo, \n\nIch habe eine Lieferung, um die ich mich kümmern muss. Die Bestellung wartet bei ~mission(Location|Address) darauf, zu ~mission(Destination|Address) gebracht zu werden. \n\nIch bin mir sicher, dass es dir gut geht, aber um ganz ehrlich zu sein, sehe ich einen Haufen Sicherheitswarnungen für dieses Gebiet auftauchen. Es scheint, dass es in den letzten Tagen eine Handvoll Angriffe in diesem Gebiet gegeben hat. Ich sage nicht, dass Ihnen etwas zustoßen wird, aber es wäre wahrscheinlich das Beste, wenn Sie ein wenig Schutz mitbringen. Logistik-Koordinator\nCovalex Shipping \n'Alles, was Sie brauchen, wo immer Sie es brauchen.'\n
```

Comment:
"Ich bin mir sicher, dass es dir gut geht" klingt in dem Zusammenhang falsch, vielleicht lieber "Ich bin sicher, dass du das schaffen wirst."